external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaRecordToUnsafeRowConverter.scala

```diff
@@ -36,7 +36,11 @@ private[kafka010] class KafkaRecordToUnsafeRowConverter {
     } else {
       rowWriter.write(0, record.key)
     }
-    rowWriter.write(1, record.value)
+    if (record.value == null) {
+      rowWriter.setNullAt(1)
+    } else {
+      rowWriter.write(1, record.value)
+    }
```

Comment:
Hi, @uncleGen . To prevent a future regression, could you add a unit test case, too?

Reply:
Please update the PR description. If `existing unit tests` already covers your case, Apache Spark don't need this PR. :)

Reply:
ok

Reply:
Thanks.